tools/src/tester/ResultsDisplayer.ts

```diff
@@ -41,7 +41,7 @@ export default class ResultsDisplayer {
 
   #display_story (): void {
     const result = this.evaluation.result
-    const message = this.evaluation.full_path
+    const message = this.evaluation.message || this.evaluation.full_path
```

Comment:
With [this](https://github.com/opensearch-project/opensearch-api-specification/pull/324/files/b0340fe5bdfac2013e63fa77cbf854b91ad16e31#r1633900477) we wont need this change anymore.